char_frequency.c

```diff
@@ -0,0 +1,46 @@
+/*Ava N.*/
+/*Write a program that takes a string of characters and returns the frequency of each character in that string.*/
+#include <stdio.h>
+#include <stdlib.h>
+
+int main(){
+        char input_string[98];       /*input string*/
+        char alpha_string[51]="abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"; 
```

Comment:
Very nicely done! You had one small warning, which caused an edge case to be missed. If the string had any capital Z's, then they weren't counted. The reason for this is because the size of the array should have been [52] instead!